EnviroMonitorStation/Config.cpp

```diff
@@ -0,0 +1,56 @@
+#include <Arduino.h>
+#include <FS.h>
+#include <ArduinoJson.h>
+#include "Config.h"
+
+void Config::mount()
+{
+  if (!SPIFFS.begin()) {
+    Serial.println("Cannot mount file system");
+  }
+}
+
+void Config::read(const char* fileName) {
+  apiEndpoint = "http://app.smogly.pl/api/v1/station/cef6960f-6627-4c4f-9080-de6bbea41a7e/add-metering/?token=9539f7d1-d633-4838-81cb-6f8c712c6ad0";
```

Comment:
It should be, we lack a proper configuration plus format (station id + token) is in flux. I'll extract it once backend will settle on a format.